### DISCUSSION TITLE
webclient query parameters

### DISCUSSION THREAD

**Discussion by JelleBuning** [rustdesk/rustdesk#4382] [Ideas]
It would be nice to automatically connect to a machine by adding an id and encrypted password queryparameter. This way it would be easy to integrate it within an existing application.

example:
https://web.rustdesk.com/?id=123123123&password=$2a$12$FE6q/Fuf6T56KzMXx5qo.eOb54FnOFXKUCNx1PGY45sr0RWZbCWx.

**Comment by TBG-FR** [comment]
That's what you did on https://github.com/JelleBuning/rustdesk/commits/fix_build/ no ? (except it's unencrypted)
